src/settings.ts

```diff
@@ -34,7 +34,8 @@ export const CREATE_EDIT_URL = "/admin";
 export const SPARKLEVERSITY_URL = "https://sparklever.se/sparkleversity";
 export const SPARKLEVERSE_COMMUNITY_URL =
   "https://www.facebook.com/groups/sparkleverse/";
-export const TICKET_URL = "https://qkt.io/seed2020";
+export const TICKET_URL = "https://sparklever.se/glowforth/";
+export const CODE_CHECK_ENABLED = true;
```

Comment:
This is quite hacky, but we needed to enable this systemwide to prevent workarounds.

Reply:
And in future this will be part of the venue's config.